Makefile

```diff
@@ -11,6 +11,7 @@ all: data/fuse/per_port_allen_pd.csv data/fuse/event_port_allen_pd.csv data/fuse
 all: data/fuse/per_madisonville_pd.csv data/fuse/event_madisonville_pd.csv data/fuse/com_madisonville_pd.csv
 all: data/fuse/per_greenwood_pd.csv data/fuse/com_greenwood_pd.csv data/fuse/event_greenwood_pd.csv
 all: data/fuse/com_new_orleans_pd.csv data/fuse/uof_new_orleans_pd.csv data/fuse/per_new_orleans_pd.csv data/fuse/event_new_orleans_pd.csv
+all: data/match/per_new_orleans_da.csv data/fuse/event_new_orleans_da.csv data/fuse/com_new_orleans_da.csv
```

Comment:
Doesn't look like there are any rule to make `data/match/per_new_orleans_da.csv`

Reply:
Ok. That makes sense. I will slow down. 

Because it was coming from a unique agency and because we have two brady lists from the New Orleans District Attorney's office (with potentially more coming), I was thinking of them as distinct. My bad.